ionc/ion_scanner.h

```diff
@@ -457,12 +457,12 @@ iERR _ion_scanner_read_as_base64                    (ION_SCANNER *scanner, BYTE
 
 iERR _ion_scanner_read_possible_number              (ION_SCANNER *scanner, int c, int sign, ION_SUB_TYPE *p_ist);
 iERR _ion_scanner_read_radix_int                    (ION_SCANNER *scanner, BYTE **p_dst, SIZE *p_remaining, int *p_char, ION_INT_RADIX radix, BOOL underscore_allowed);
-iERR _ion_scanner_read_hex_int                      (ION_SCANNER *scanner, BYTE **p_dst, SIZE *p_remaining);
-iERR _ion_scanner_read_binary_int                   (ION_SCANNER *scanner, BYTE **p_dst, SIZE *p_remaining);
+iERR _ion_scanner_read_hex_int                      (ION_SCANNER *scanner, BYTE **p_dst, SIZE *p_remaining, int *p_char);
```

Comment:
Instead of `p_char`, would something like `p_end_char` be more descriptive of what this parameter is used for?

Reply:
Sure.

Reply:
Looks like you missed `_ion_scanner_read_timestamp`.

Reply:
Added.